packages/jest-util/src/index.ts

```diff
@@ -6,9 +6,11 @@
  */
 
 // need to do this for api-extractor: https://github.com/microsoft/rushstack/issues/2780
+import * as ansiEscapes from './ansiEscapes';
 import * as preRunMessage from './preRunMessage';
 import * as specialChars from './specialChars';
 
+export {ansiEscapes};
```

Comment:
One more idea. Additionally `jest-watcher` could also expose `ansiEscapes` so that plugins could use this implementation as well.